### PR TITLE
catalog was created

### PR DESCRIPTION
**Pull request by vanyachyzh**
- [DEMO LINK](https://vanyachyzh.github.io/layout_catalog/)
- [TEST REPORT LINK](https://vanyachyzh.github.io/layout_catalog/report/html_report/)

1. I use https://github.com/import url instead just https://github.com/import because it doesn't work for me.
2. There is a problem with test Main_tag_1200px.png but I don't understand what the problem is.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,298 @@
...
+            <a href="#" class="nav-list__link">Photo</a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content__cards-wrapper">
```
Comment (mykhalenych) [inline src/index.html:55]: it isn't bem methodology, block = content, element = cards, modificatory = wrapper it isn't correct

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,298 @@
...
+      <a class="img-link" href="#home">
+        <img 
+          src="./images/logo.png" 
+          alt="Logo" 
+          class="logo"
+        >
+      </a>
+      
+      <nav>
+        <ul class="nav-list">
```
Comment (mykhalenych) [inline src/index.html:20]: not correct naming
https://en.bem.info/methodology/html/

Comment (mykhalenych) [reply src/index.html:20]: where is class nav?

File: src/index.html [hunk 3]
```diff
@@ -1,12 +1,298 @@
...
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content__cards-wrapper">
+      <div class="product-card content__product-card" data-qa="card">
```
Comment (mykhalenych) [inline src/index.html:56]: the same

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,77 @@
+.product-card {
+  font-style: normal;
+  color: #060b35;
```
Comment (mykhalenych) [inline src/styles/blocks/card.scss:3]: you use this color several times, make a variable and import

File: src/styles/reset.scss [hunk 5]
```diff
@@ -0,0 +1,17 @@
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+p {
```
Comment (mykhalenych) [inline src/styles/reset.scss:9]:
```
body,
p,
h4 {
  margin: 0;
  padding: 0;
}
```

**Review by nazarmatsevych — APPROVED** [review]

**Review by nazarmatsevych — APPROVED** [review]

**Pull request closed without merging**